app/src/androidTest/java/at/tugraz/ist/guessingwords/data/DatabaseTest.kt

```diff
@@ -108,4 +108,14 @@ class DatabaseTest {
         assert(word.text == newWord.text)
         assert(ruid == newWord.uid)
     }
+
```

Comment:
Maybe write a test that looks at the in memory instance? If it was created, if it is different from the normal db, stuff like that. Should be very fast to write.